### DISCUSSION TITLE
Set Device Endpoint Cluster Attribute via MQTT

### DISCUSSION THREAD

**Discussion by ckglobalroaming** [Koenkk/zigbee2mqtt#24897] [Devices]
### **QUESTION:**
Can we set Device Endpoint Cluster config via MQTT?

### **PROBLEM STATEMENT:**
I have Zigbee PA-44Z Smoke detectors. I have configured their reporting interval by setting a generic report as follows.

`Endpoint:1 / Cluster:GenBasic / Attribute: modelId / Min Rep Interval: 120 / Max Rep Interval: 120 / Min Rep Change: 0`

This successfully makes the tuya smoke detector send a payload every two minutes. That payload is a full status so all the other metrics get updated in Z2M and then to Homeassistant.

**RESETS TO DEFAULTS:**
If the smoke detector is reset or rejoins the network then the  above reporting config dissapears as it had left the network and the reporting returns to that set by the interview.

### **SOLUTION REQUEST:**
If we can set the device endpoint reporting metrics by way of an MQTT payload publish, then I could reconfigure the device reporting interval by way of a Homeassistant Automation.

### **CLOSING**
Any guidance if this is possible or consideration for the code implementation would be appreciated. If i hear back that this needs to be a feature request then I will raise one for consideration..

Thanks.

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request-device-configure-reporting
